appsec/src/extension/tags.c

```diff
@@ -1423,6 +1433,110 @@ static PHP_FUNCTION(datadog_appsec_track_user_login_failure_event)
     dd_tags_set_sampling_priority();
 }
 
+static PHP_FUNCTION(datadog_appsec_track_authenticated_user_event_automated)
+{
+    UNUSED(return_value);
+    if (!DDAPPSEC_G(active)) {
+        mlog(dd_log_debug, "Trying to access to track_user_login_failure_event "
+                           "function while appsec is disabled");
+        return;
+    }
+
+    zend_string *user_id;
+    zend_string *anon_user_id = NULL;
+    if (zend_parse_parameters(ZEND_NUM_ARGS(), "S", &user_id) == FAILURE) {
+        mlog(dd_log_warning,
+            "Unexpected parameter combination, expected user_id");
+        return;
+    }
+
+    if (ZSTR_LEN(user_id) == 0) {
+        mlog(dd_log_warning, "Unexpected empty user id");
```

Comment:
Since this is an automated integration, I'm not sure if `warning` is the right approach as we may flood logs

Reply:
Switched the automated logs to debug